governance-exercises/VoteWizard_executive_exercise_02-08-2023/Executive vote - August 2, 2023.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Template - [Executive Vote] Enhanced Dai Savings Rate Activation, Spark Protocol D3M Parameters Updates, Harbor Trade Credit and New Silver RWA Vault Updates, Q2 2023 AVC Members Compensation, DAO Resolution for Monetalis Clydesdale, Launch Project Funding, Spark Proxy Spell Execution - August 2, 2023
+summary: Increase the Dai Savings Rate to 8% with the activation of the Enhanced Dai Savings Rate, increase the Maximum Debt Ceiling for the Spark Protocol D3M, decrease the Debt Ceiling for Harbor Trade Credit (RWA004-A), update the Doc parameter for New Silver (RWA002-A), compensate AVC Members for the second quarter of 2023, distribute funding for the Launch Project, execute a proxy spell for Spark Protocol.
+date: 2023-08-02T00:00:00.000Z
+address: "0x73603dB34814b22379CeD3d2Cbb450B3968Fd892"
+
+---
+# [Executive Proposal] Enhanced Dai Savings Rate Activation, Spark Protocol D3M Parameters Updates, Harbor Trade Credit and New Silver RWA Vault Updates, AVC Member Compensation for Q2 2023, DAO Resolution for Monetalis Clydesdale, Launch Project Funding, Spark Proxy Spell Execution - August 2, 2023
+
+The Governance Facilitators, Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+- The Enhanced Dai Savings Rate will be activated increasing Dai Savings Rate to 8%, as detailed below.
+- The Spark Protocol Dai Direct Deposit Module (D3M) Maximum Debt Ceiling will be increased, as detailed below.
+- The Harbor Trade Credit (RWA004-A) vault Debt Ceiling will be decreased and will subsequently transition to a soft liquidation state, as detailed below.
+- The New Silver (RWA002-A) RWA Agreement (doc) parameter will be updated, as detailed below.
+- The AVC Member Compensation for the Second Quarter of 2023 will be carried out, as detailed below.
```

Comment:
You can include the MKR total here fairly easily. Maybe rephrase as well, wording is  a little awkward.